landing/src/app/globals.css

```
@@ -129,15 +177,32 @@
+  --color-outline-variant: var(--outline-variant);
+
+  /* ── Radius: sharp corners, max 2px ── */
+  --radius-xl: 6px;
+  --radius-lg: 8px;
+  --radius-md: 4px;
+  --radius-sm: 2px;
 
   /* ── Typography ── */
   --font-sans: "Inter", ui-sans-serif, system-ui, sans-serif;
   --font-heading: "Space Grotesk", ui-sans-serif, system-ui, sans-serif;
   --font-mono: "Space Mono", ui-monospace, SFMono-Regular, Consolas, monospace;
```

Stylelint (17.7.0)

[error] Expected "SFMono-Regular" to be "sfmono-regular" (value-keyword-case)

(value-keyword-case)